### ISSUE TITLE
Feedback: 'Integrating with Azure Application Insights'

### ISSUE THREAD

**Issue by bbrandt** [Particular/docs.particular.net#6446]
Feedback for 'Integrating with Azure Application Insights' https://docs.particular.net/servicecontrol/app-insights-integration

Location in GitHub: https://github.com/Particular/docs.particular.net/blob/master/servicecontrol/app-insights-integration.md

This statement mentions retries, processing time, and critical time:
> It is possible to gather performance metrics like [Processing Time](https://github.com/Particular/docs.particular.net/blob/master/monitoring/metrics/definitions.md#metrics-captured-processing-time) and [Critical Time](https://github.com/Particular/docs.particular.net/blob/master/monitoring/metrics/definitions.md#metrics-captured-critical-time) as well as data on the number of immediate and delayed [retries](https://github.com/Particular/docs.particular.net/blob/master/nservicebus/recoverability).

But this links to a sample that does not seem to publish retries, processing time, and critical time:
> For more details, see [this sample](https://github.com/Particular/docs.particular.net/blob/master/samples/open-telemetry/application-insights).

List of metrics from the sample:
> Metrics:
nservicebus.messaging.successes
nservicebus.messaging.failures
nservicebus.messaging.fetches

**Comment by DavidBoike** [comment]
Thanks Ben,

We're consolidating some samples in anticipation of the release of NServiceBus 9 and it looks like the task force got a little over-zealous and redirected the old sample that talked about all those things to the newer OpenTelemetry sample that doesn't quite do all those things. We're going to look at bringing those concepts over to the OTel sample to have a best of both worlds. Stay tuned.

**Comment by andreasohlund** [comment]
@bbrandt FYI we are working on the expanded sample over here if you are curious

https://github.com/Particular/docs.particular.net/pull/6452

**Comment by andreasohlund** [comment]
@SzymonPobiega @lailabougria, did we add the mentioned metric natively in v9.1?

**Comment by lailabougria** [comment]
We did: https://docs.particular.net/nservicebus/operations/opentelemetry#meters-emitted-meters

**Comment by andreasohlund** [comment]
@lailabougria so we can go through the v9 samples and remove the shims right? 

eg. https://github.com/Particular/docs.particular.net/tree/master/samples/open-telemetry/application-insights/Core_9/Endpoint/MetricsShim

**Comment by SzymonPobiega** [comment]
Yes, we are doing it as part of https://github.com/Particular/docs.particular.net/pull/6777
